### DISCUSSION TITLE
Local thermal nonequilibrium heat transfer in porous media

### DISCUSSION THREAD

**Discussion by Bearunner** [idaholab/moose#28080] [Q&A Modules: General]
Dear all,

Do we have modules for simulating local thermal nonequilibrium heat transfer (Trock not equal to Tfluid) in porous media?

Thanks a lot.

B

**Comment by GiudGiud** [comment]
@cpgr 

I suspect we can do that, we can solve transient simulation with a dynamic temperature field.

**Reply by Bearunner** [reply]
> @cpgr
> 
> I suspect we can do that, we can solve transient simulation with a dynamic temperature field.

Hello Guillaume, thanks for your answer. 

I find that all the energy/heat kernels in Porousflow Modules are based on the equilibrium heat transfer theory.

**Reply by cpgr** [reply]
Yeah, I don't think this has been added yet.
